invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsModal.js

```diff
@@ -300,12 +301,20 @@ export class CreatibutorsModal extends Component {
         extra: creatibutor,
         key: creatibutor.id,
         content: (
-          <Header>
-            {creatibutor.name} {idString.length ? <>({idString})</> : null}
-            <Header.Subheader>
-              {isOrganization ? creatibutor.acronym : affNames}
-            </Header.Subheader>
-          </Header>
+          <Overridable
+            id="CreatibutorsModal.RemoteSelectItem.content"
```

Comment:
does it make sense to follow [our conventions](https://inveniordm.docs.cern.ch/develop/best-practices/react/#overridable-components-naming) for the naming here?